portal/static/js/initialQueries.js

```diff
@@ -699,14 +699,14 @@
         var termsEvent = function() {
             if ($(this).attr("data-agree") === "false") {
                 var types = $(this).attr("data-tou-type");
-                if (hasValue(types)) {
+                if (types) {
```

Comment:
This function gets pretty deeply indented. You could remove a layer of indentation if you inverted this into a guard clause and exited early if that condition isn't met.